plugins/osdn/common.go

```diff
@@ -214,10 +214,27 @@ func (oc *OvsController) StartNode(mtu uint) error {
 		return fmt.Errorf("Failed to start plugin: %v", err)
 	}
 
-	if oc.ready != nil {
-		close(oc.ready)
+	oc.MarkPodNetworkReady()
```

Comment:
MarkPodNetworkReady() doesn't need to be accessed from outside the OVSController, right?  Maybe we should lower-case the 'M' then so it's not exported.